auth-web/src/models/address.ts

```diff
@@ -1,10 +1,20 @@
 export interface Address {
-    city?: string
-    country?: string
-    street?: string
-    streetAdditional?: string
-    postalCode?: string
-    region?: string
-    key?: string
+  city?: string
+  country?: string
+  street?: string
+  streetAdditional?: string
+  postalCode?: string
+  region?: string
+  deliveryInstructions?: string
+  key?: string
+}
 
+export interface IAddress {
```

Comment:
just curious , whats the I in IAddress ... may be find a better name...not super picky on this..if possible